### ISSUE TITLE
Syntax error in React wrapper compilation

### ISSUE THREAD

**Issue by m-clare** [yairEO/tagify#287]
### Explanation

I'm unable to compile my component using the react distribution of tagify due to a syntax error in ``react.tagify.js``

Using the [sample react component](https://codepen.io/vsync/project/editor/ARaJye), my imports are:

```import React, { Component } from 'react';```
```import Tags from '@yaireo/tagify/dist/react.tagify'```

This is unable to compile and I get the following error:

```Failed to compile.

./node_modules/@yaireo/tagify/dist/react.tagify.js
SyntaxError: /Users/mclare/workspaces/engage/node_modules/@yaireo/tagify/dist/react.tagify.js: Unexpected token (45:12)

  43 | 
  44 |         return this.props.mode === 'textarea' ?
> 45 |             <textarea {...attrs} defaultValue={this.props.initialValue}></textarea> :
     |             ^
  46 |             <input {...attrs} defaultValue={this.props.initialValue} />
  47 |     }
  48 | }

**Comment by yairEO** [comment]
I will check today and let you know, thanks for reporting!

**Comment by yairEO** [comment]
Please provide a demo link if possible because in *Codepen* it works, therefor it must be something specific to your implementation

**Comment by yairEO** [comment]
Fixed, try with version `2.26.0`

**Comment by m-clare** [comment]
It works! Thank you!
